### PR TITLE
upgrade dependency

### PR DESCRIPTION
**Pull request by hero78119**

### CODE REVIEW

File: snark-verifier/src/loader/halo2/shim.rs [hunk 1]
```diff
@@ -272,10 +272,11 @@ mod halo2_wrong {
                     )?;
                     let acc =
                         Value::known(*scalar) * lhs.value() * rhs.value() + Value::known(constant);
-                    let output = values.iter().skip(1).fold(
```
Comment (hero78119) [inline snark-verifier/src/loader/halo2/shim.rs:275]: clippy fix: use `try_fold`

File: snark-verifier/src/loader/halo2/shim.rs [hunk 2]
```diff
@@ -419,10 +420,7 @@ mod halo2_wrong {
                     Ok::<_, Error>((scalar.deref().clone(), self.assign_constant(ctx, *base)?))
                 })
                 .collect::<Result<Vec<_>, _>>()?;
-            let pairs = pairs
```
Comment (hero78119) [inline snark-verifier/src/loader/halo2/shim.rs:422]: clippy fix: complain removing `map` but we still need ref-of-tuple to tuple-of-ref

File: snark-verifier/src/pcs/kzg/accumulation.rs [hunk 3]
```diff
@@ -46,7 +46,7 @@ where
         let (lhs, rhs) = instances
             .iter()
             .map(|accumulator| (&accumulator.lhs, &accumulator.rhs))
-            .chain(proof.blind.as_ref().map(|(lhs, rhs)| (lhs, rhs)))
```
Comment (hero78119) [inline snark-verifier/src/pcs/kzg/accumulation.rs:49]: diddo

**Review by han0110 — APPROVED** [review]
LGTM!

**Pull request merged**